.cirrus.yml

```diff
@@ -928,7 +939,17 @@ minikube_test_task:
     name: *std_name_fmt
     alias: minikube_test
     # Docs: ./contrib/cirrus/CIModes.md
-    only_if: *not_tag_magic
+    # only when: - it is not a PR (we never want to skip on nightly tests); or
+    #            - CI:ALL title is set; or
+    #            - danger files are changed; or
+    #            - minikube test code is changed; or
+    #            - source code is changed AND NOT only test files
+    only_if: >-
+        $CIRRUS_PR == '' ||
+        $CIRRUS_CHANGE_TITLE =~ '.*CI:ALL.*' ||
+        changesInclude('.cirrus.yml', 'Makefile', 'contrib/cirrus/**', 'vendor/**', 'hack/**', 'version/rawversion/*') ||
+        changesInclude('test/minikube/**') ||
```

Comment:
see note about `system`. (Not too important because this task is very likely to be eliminated soon)